packages/mobile/src/exchange/__snapshots__/ExchangeTradeScreen.test.tsx.snap

```diff
@@ -329,7 +348,7 @@ exports[`ExchangeTradeScreen renders correctly 1`] = `
             >
               
               ₱
-              12.09
+              181.81
```

Comment:
just checking, is this new value expected?

Reply:
Yes, it surprised me too, along with the loading above being removed. I think it was due to how the test was loading stuff before but doing the math this looks correct.

This is the exchange rate between cUSD and CELO.

To get this, first we need to convert CELO to cUSD and then cUSD to PHP.

CELO to cUSD is `9.09090909` and cUSD to PHP is '20'
So 1 CELO = `₱181.8181818`

No clue where the 12.09 came from, but 181 is the value I would expect here